types/index.d.ts

```diff
@@ -0,0 +1,59 @@
+// TypeScript Version: 3.8
```

Comment:
Bumped this from 3.1 to 3.8 for this support:
https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-8.html#type-only-imports-and-exports